front/lib/api/assistant/configuration.ts

```diff
@@ -118,6 +118,7 @@ function determineGlobalAgentIdsToFetch(
     case "list":
     case "all":
     case "admin_internal":
+    case "manage-assistants-search":
       // All global agents in global, list, all, admin_internal views.
```

Comment:
I guess this comment is now outdated (but the comment is redundant so could be removed IMO)